boa/profiling.py

```diff
@@ -225,6 +212,43 @@ def get_line_data(self):
         return line_gas_data
 
 
+# singleton profile object which collects gas+line profiles over test runs
```

Comment:
This could become a memory issue for large test suites

Reply:
true, but such is the nature of profiling. it's also not a regression, since the same issue existed before (only it was stored on the `Env()`, so the issue was more difficult to diagnose)